### ISSUE TITLE
Remove all the unused and useless graphs from dashboard.

### ISSUE THREAD

**Issue by sirinoks** [Seneca-CDOT/telescope#3034]
Turns out all these cool looking graphs are not doing anything. They are a left over from the time when someone yoinked the design. Not just the parts that we needed, but the whole thing!
![Screenshot_480](https://user-images.githubusercontent.com/13803161/155542438-a9d6d1af-eb4d-4c9f-9ae8-2e0c3e663b4c.png)
 
You might want to confirm which parts are exactly useless, so correct my issue if I'm wrong to circle those. 

Basically you need to delete some code and make sure everything still works.

**Comment by pbelokon** [comment]
I would like to work on it.

**Comment by humphd** [comment]
@pbelokon welcome!  Let us know what help you need.  I'm sure @dbelokon would be happy to provide support!
